manifests/admin.pp

```diff
@@ -184,4 +205,13 @@
     priority => 10,
     content  => $_content,
   }
+
+  if $set_selinux_login {
+    if $facts['selinux_current_mode'] and ($facts['selinux_current_mode'] != 'disabled') {
+      selinux_login { "%${admin_group}":
```

Comment:
there is no test in the ``selinux`` module for  ``selinux_login`` with a %<group>.  Is this a problem?

Reply:
@lnemsick-simp It's just an arbitrary string. If that doesn't work, then the `man` page is wrong.